app/controllers/cart_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CartController < ApplicationController
+  def create
+    if Cart.last
```

Comment:
It's similar to the comment I made earlier. 

Since you're using the if statement to decide what value to assign the variable `cart` here it wants you to do something like

```ruby
cart = if Cart.last
             Cart.last
           else
             Cart.create
          end
```

But the previous one I showed you is probably cleaner IMO